src/fides/api/models/privacy_request.py

```diff
@@ -1653,7 +1673,11 @@ class ExecutionLog(Base):
         nullable=False,
     )
     status = Column(
-        EnumColumn(ExecutionLogStatus),
+        EnumColumn(
+            ExecutionLogStatus,
+            native_enum=True,
+            values_callable=lambda x: [i.value for i in x],
```

Comment:
This ensures we're still using the enum values in the database, so still using "paused" even though the application is calling it "awaiting_processing".

Reply:
Clever! ✨